Wikipedia/Code/UIImageView+WMFImageFetchingInternal.m

```diff
@@ -187,6 +190,14 @@ - (void)wmf_setImage:(UIImage *)image
     };
     
     self.image = image;
+    
+    if (image.isGIF && [self isKindOfClass:[FLAnimatedImageView class]]) {
+        NSURL *imageURL = [self wmf_imageURLToFetch];
+        FLAnimatedImage *animatedImage = [FLAnimatedImage animatedImageWithGIFData: [[self wmf_imageController] diskDataForImageWithURL:imageURL]];
+        FLAnimatedImageView *animatedImageView = ((FLAnimatedImageView*) self);
```

Comment:
There's a potential that self isn't actually an `FLAnimatedImageView` - this should be verified with `if ([self isKindOfClass:[FLAnimatedImageView class]])`